docs/pipeline/misc-portal-lambda.md

```diff
@@ -0,0 +1,37 @@
+# Misc Portal Lambda
+
+### Fastq lambda
```

Comment:
The preferred way is to go through the API. 
Do you want to document this? Do we still have usage of this?

Reply:
Removing